gateway/1.8/modules/ROOT/pages/flex-review-prerequisites.adoc

```diff
@@ -99,6 +99,31 @@ A single Flex Gateway can support multiple backend APIs. To support more backend
 
 Flex Gateway requires either an Intel or AMD-64 processor.
 
+[[limits]]
+== Limits
+
+[%header%autowidth.spread,cols="a,>.<a,a"]
+|===
+| Limit | Value | Notes
+| APIs per gateway instance
+| {gateway-api-max} 
+| Maximum supported.
```

Comment:
It's a little unclear what you mean by "Maximum supported". The whole table is defining the limits, which would imply the maximum supported, I think? 

I thought maybe there were two limits? If so, then I would expect the Notes column to include a number. Let me find an example. 


Suggest making this a sentence indicating this is the maximum supported value. Or supported maximum?

Reply:
I can't find any examples :(

Reply:
This limit (Value column) is the maximum supported value. There is no hard / error limit for these rows. I couldn't find an example either but I can send you another proposal to make it more clear.

Reply:
Then, I don't think you need it in the Notes column. 

You could add an intro the table, explaining that...

Reply:
How about two tables?
https://github.com/mulesoft/docs-gateway/pull/673/files

Reply:
OK, I reordered the limits to put hard error limits first, and I change the note for supported limits to say:

> Exceeding this limit will will not cause errors, but is not supported.